libportal/meson.build

```diff
@@ -58,7 +58,7 @@ install_headers(headers, subdir: 'libportal')
 libportal = library('portal',
                     src,
                     version: '0.0.1',
```

Comment:
This should be `1.0.0`.

You can actually remove `soversion` entirely as the first number of `version` is used.

Reply:
Errr, apologies for the massive brainfart with the library version. Should be good to go now.